azurerm/internal/acceptance/plugin_sdk_aliases.go

```diff
@@ -0,0 +1,59 @@
+package acceptance
```

Comment:
Wondering why this file is not placed to the `tfpluginsdk` folder as others.

Reply:
these are purely test related (and in this instance a tie-over) - so whilst we _could_ put these in the `pluginsdk` folder, having all of the test items in a single place is itself helpful for discovery (even if the methods in this file are mostly temporary, as a general rule this means we can document the `acceptance` package and how to use it)

Reply:
for now I'm going to merge this since it's blocking several other things - I don't think we should move the test related items out (since they're explicitly test related and will be deprecated in favour of the other test bits) - but the other ones I'm open to splitting out into nested packages presuming we can group these adequately